Update index.rst (#4)
qiime2/paper2: sphinx_docs/index.rst

.. command-block::

>     for f in `ls per_sample_FASTQ/81253/*.gz`; do n=`basename $f`; echo -e "12802.${n%.fastq.gz}\t$PWD/$f"; done >> manifest.tsv

Reviewer: ```suggestion
    for f in `ls per_sample_FASTQ/81253/*.gz`; do n=`basename $f`; echo -e "12802.${n/.fastq.gz}\t$PWD/$f"; done >> manifest.tsv
```

   :no-exec:

    conda install -c bioconda bowtie2
    conda install cytoolz
    pip install https://github.com/knights-lab/SHOGUN/archive/master.zip
    pip install https://github.com/qiime2/q2-shogun/archive/master.zip
    qiime dev refresh-cache

2. Download all the required example files from the q2-shogun repository:

> .. command-block::

Reviewer: ```suggestion
.. download::
   :url: https://github.com/qiime2/q2-shogun/raw/master/q2_shogun/tests/data/query.qza
   :saveas: query.qza

.. download::
   :url: https://github.com/qiime2/q2-shogun/raw/master/q2_shogun/tests/data/refseqs.qza
   :saveas: refseqs.qza

.. download::
   :url: https://github.com/qiime2/q2-shogun/raw/master/q2_shogun/tests/data/taxonomy.qza
   :saveas: taxonomy.qza
```

>     for i in query refseqs taxonomy bt2-database; do wget https://github.com/qiime2/q2-shogun/raw/master/q2_shogun/tests/data/$i.qza; done

Reviewer: ```suggestion
.. download::
   :url: https://github.com/qiime2/q2-shogun/raw/master/q2_shogun/tests/data/bt2-database.qza
   :saveas: bt2-database.qza
```

In this example, SHOGUN is called to align query sequences ``query.qza``
against a reference sequence database refseqs.qza using the popular short
sequence aligner Bowtie2 (Langmead & Salzberg, 2012). The query sequences may
be demultiplexed or multiplexed data. In the latter case, SHOGUN will
automatically stratify alignment results by sample ID. The taxonomy artifact
``taxonomy.qza`` defines the mapping of reference sequences to taxonomic
lineages. In addition to taxonomy, this artifact could be any hierarchical
(semicolon-delimited) or simple mappings, for example, functional annotations. A

> Bowtie2 index containing the reference sequence databaseis necessary for this operation.

Reviewer: ```suggestion
Bowtie2 index containing the reference sequence database is necessary for this operation.
```
